### PR TITLE
README update to add app title

### PR DESCRIPTION
**Pull request by abbodh**
Added application title name

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ urlFragment: microsoft-teams-apps-quickresponses
 | [Documentation](https://github.com/OfficeDev/microsoft-teams-apps-quickresponses/wiki/Home) | [Deployment guide](https://github.com/OfficeDev/microsoft-teams-apps-quickresponses/wiki/Deployment-Guide) | [Architecture](https://github.com/OfficeDev/microsoft-teams-apps-quickresponses/wiki/Solution-Overview) |
 | ---- | ---- | ---- |
 
+## Quick Responses app
```
Comment (aosolis) [inline README.md:14]: Make this #
And change it to "Quick Responses App Template"
(same for the other repos)

File: README.md [hunk 2]
```diff
@@ -28,7 +30,7 @@ Please read the license terms applicable to this app template [here](https://git
 
 * Microsoft will have no access to data collected through your app.  Microsoft will not be responsible for any data related incidents or data subject requests.
 
-* Any trademarks or registered trademarks of Microsoft in the United States and/or other countries and logos included in this repository are the property of Microsoft, and the license for this project does not grant you rights to use any Microsoft names, logos or trademarks outside of this repository.  Microsoft�s general trademark guidelines can be found [here](https://www.microsoft.com/en-us/legal/intellectualproperty/trademarks/usage/general.aspx)
+* Any trademarks or registered trademarks of Microsoft in the United States and/or other countries and logos included in this repository are the property of Microsoft, and the license for this project does not grant you rights to use any Microsoft names, logos or trademarks outside of this repository.  Microsoft’s general trademark guidelines can be found [here](https://www.microsoft.com/en-us/legal/intellectualproperty/trademarks/usage/general.aspx)
```
Comment (aosolis) [inline README.md:33]: What did you change here?

File: README.md (hunk 1 above)
Comment (abbodh) [reply README.md]: done, made the same change in other repos.

File: README.md (hunk 2 above)
Comment (abbodh) [reply README.md:33]: weird. it didn't showed up in GitHub and IDE as difference. I believe the encoding got changed to UTF-8 

From git diff
**Microsoft's** text before Microsoft<92>s changed to Microsoft<E2><80><99>s. No changes in actual text.

**Pull request merged**